### DISCUSSION TITLE
Running the ManagedWindow example and the ZQM issue

### DISCUSSION THREAD

**Discussion by HoWeBrz** [pymeasure/pymeasure#591] [Q&A]
Hi, 
I tried to run the ManagedWindow example [https://pymeasure.readthedocs.io/en/latest/tutorial/graphical.html#using-the-managedwindow](url), but still see the ZQM issue [https://github.com/pymeasure/pymeasure/issues/168](url).

I am using pyMeasure0.9 with Python3.9 and applied the proposed workaround
```
import sys
sys.modules['cloudpickle'] = None
```
Removing cloudpicle is not an option, since I am using Spyder as editor, and it requires cloudpickle.

I want to use pyMeasure in our lab and either want the workaround running, or checkout the latest releases from GIT.
Since my experience with GIT is nil, I'd rather use the workaraound first, and later try GIT.

Can you help me in getting the proposed workaround to work ?

Thx
Ho

**Comment by bilderbuchi** [comment]
You don't necessarily need Git experience. If you are using pip (and not, say, conda), you can use its support for [version control systems](https://pip.pypa.io/en/stable/topics/vcs-support/#git) to pip-install pymeasure from current master.
Did not try this, but ` pip install git+https://github.com/pymeasure/pymeasure.git` should probably "just work".

**Reply by HoWeBrz** [reply]
Hi,
I did pull the latest git-version via tortoise-SVN (since I have it installed already), and found that the issue in the latest git-version is resolved.
I am wondering why the workaround `sys.modules['cloudpickle'] = None`   does not work.

Its unrelated to spyder - the command seems to have no effect at all.
Do you have any idea ?

**Reply by bilderbuchi** [reply]
No, sorry. Glad that your problem is fixed!
